modules/vpc-and-vpn/vpn.tf

```diff
@@ -13,7 +14,7 @@ locals {
     openvpn_routes         = [{ "network" : cidrhost(var.network_cidr, 0), "netmask" : cidrnetmask(var.network_cidr) }],
     openvpn_dns_servers    = [cidrhost(var.network_cidr, 2)], # The second ip is the DNS in AWS
     openvpn_dhparam_bits   = var.vpn_dhparams_bits,
-    furyagent_version      = "v0.2.2"
+    furyagent_version      = "v0.3.0"
```

Comment:
beware that changing this value has an impact on existing projects. Changing this value changes the cloudinit configuration for the bastions and so triggering a destroy&create in terraform of the instances, and they lose the associated EIP, breaking the client's VPN configuration.

See #49

Reply:
Solved in 26188c7810ca575515d0671a4fbbc47157b1ea8b Ignore changes in cloudinit configuration.

Reply:
Another solution could be to create public interfaces with associated public IP and then connect them to instances.

Reply:
Ignoring changes to `userdata` will break other features like the ssh users, won't it? IMO the right way to do it is to decouple the EIP lifecycle form the instance and attach them later.

Reply:
Ignoring changes to user_data will ignore any change to the following variables:
https://github.com/sighupio/fury-eks-installer/blob/c9b845dd4c5d13205af0f0c9d5f92c1a7cc6df8b/modules/vpc-and-vpn/vpn.tf#L9-L18
do we want that? for example, If I change for example the VPN DNS server (after having created the bastion) that change won't be applied with the ignore.

If the EIP is already decoupled, what do we need to do so it doesn't change when the bastion is recreated?

Reply:
https://github.com/sighupio/fury-eks-installer/commit/26188c7810ca575515d0671a4fbbc47157b1ea8b dropped.

Now, each time the `user data` is changed, the VPN hosts are recreated, attaching the previous EIP.